alephclient/tasks/crawldir.py

```diff
@@ -69,14 +78,14 @@ def crawl_dir(api, path, foreign_id, config):
     foreign_id: foreign_id of the collection to use.
     language: language hint for the documents
     """
-    path = Path(path).resolve()
+    p = Path(path).resolve()
```

Comment:
Nooo single letter variables, why? :)

Reply:
So, it is a little bit stupid, but changing the type of a variable on the fly makes the typing validation unhappy. Would you be ok with `_path` instead?

Reply:
Oh, also, in the same method, you have a variable called `q` :p